wallet/wallet-cli-lib/src/commands/mod.rs

```diff
@@ -888,6 +911,64 @@ impl CommandHandler {
                 Self::broadcast_transaction(rpc_client, tx).await
             }
 
+            WalletCommand::CreateDelegation { address, pool_id } => {
+                let address = parse_address(chain_config, &address)?;
+
+                let _status = controller_opt
+                    .as_mut()
+                    .ok_or(WalletCliError::NoWallet)?
+                    .create_delegation(
+                        selected_account.ok_or(WalletCliError::NoSelectedAccount)?,
+                        address,
+                        pool_id.take(),
+                    )
+                    .await
+                    .map_err(WalletCliError::Controller)?;
+                Ok(ConsoleCommand::Print("Success".to_owned()))
```

Comment:
Instead of these obscure "Success" messages, I really prefer to write more information. Something like "Success. The transaction was broadcast to the network.". We should do the same for the GUI.